tools/c7n_azure/c7n_azure/resources/load_balancer.py

```diff
@@ -24,9 +25,28 @@ class resource_type(object):
         client = 'NetworkManagementClient'
         enum_spec = ('load_balancers', 'list_all')
         id = 'id'
+        type = 'loadbalancer'
         name = 'name'
         default_report_fields = (
             'name',
             'location',
             'resourceGroup'
         )
+
+@LoadBalancer.filter_registry.register('frontendip')
```

Comment:
I'm not sure of the exact naming rules but when I was naming my last one I noticed that a search in the repo for `filter_registry.register` shows they universally use dashes (including on relatedResourceFilters), so this would be `frontend-ip` I think.   Since the related azure object is publicIP actually should be `public-ip` perhaps.  (and in the schema definition below)

Reply:
Thanks! Renamed to frontend-public-ip